src/mesh/RadioInterface.cpp

```diff
@@ -169,6 +169,14 @@ const RegionInfo regions[] = {
     */
     RDEF(KZ_433, 433.075f, 434.775f, 100, 0, 10, true, false, false), RDEF(KZ_863, 863.0f, 868.0f, 100, 0, 30, true, false, true),
 
+
+      /*
+        Nepal
+        865 MHz to 868 MHz frequency band for IoT (Internet of Things), M2M (Machine-to-Machine), and smart metering use, specifically in non-cellular mode.
+        https://www.nta.gov.np/uploads/contents/Radio-Frequency-Policy-2080-English.pdf
+    */
+    RDEF(NP_865, 865.0f, 868.0f, 100, 0, 30, true, false, false)
```

Comment:
```suggestion
    /*
        Nepal
        865 MHz to 868 MHz frequency band for IoT (Internet of Things), M2M (Machine-to-Machine), and smart metering use, specifically in non-cellular mode.
        https://www.nta.gov.np/uploads/contents/Radio-Frequency-Policy-2080-English.pdf
    */
    RDEF(NP_865, 865.0f, 868.0f, 100, 0, 30, true, false, false),
```
Failing to compile without this `,`